packages/graphql_flutter/lib/src/widgets/mutation.dart

```diff
@@ -1,13 +1,10 @@
 import 'package:flutter/widgets.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
 
 import 'package:graphql/client.dart';
+import 'package:graphql_flutter/src/widgets/hooks/mutation.dart';
 
-import 'package:graphql_flutter/src/widgets/graphql_provider.dart';
-
-typedef RunMutation<TParsed> = MultiSourceResult<TParsed> Function(
-  Map<String, dynamic> variables, {
-  Object? optimisticResult,
-});
+export 'package:graphql_flutter/src/widgets/hooks/mutation.dart';
```

Comment:
same question here, sure that I'm missing somethings

Reply:
Good spot! I've streamlined the imports!